flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/Executor.java

```diff
@@ -46,7 +46,7 @@ static Executor create(
      *
      * @return the session configuration.
      */
-    ReadableConfig getSessionConfig();
+    Map<String, String> getSessionConfig();
```

Comment:
Is this a public api? Although this is not clearly annotated as 'api', we may need to be careful about this change. CC @fsk119 what do you think about this since you are the original author of this class.

Reply:
Yes. I think external system may rely on this. Can we add another method, e.g. getSessionConfigMap?

Reply:
DONE